dame_flame/matching.py

```diff
@@ -146,30 +146,29 @@ def predict(self, input_data):
             input_data, self.holdout_data)
 
         self.return_array = _DAME(self.input_data.copy(deep=True),
-            self.holdout_data.copy(deep=True),
-            self.treatment_column_name, self.weight_array,
-            self.outcome_column_name, self.adaptive_weights, self.alpha,
-            self.repeats, self.verbose, self.want_pe,
-            self.early_stop_iterations, self.stop_unmatched_c,
-            self.early_stop_un_c_frac, self.stop_unmatched_t,
-            self.early_stop_un_t_frac, self.early_stop_pe,
-            self.early_stop_pe_frac, self.want_bf,
-            self.missing_indicator,
-            self.missing_data_replace, self.missing_holdout_replace,
-            self.missing_holdout_imputations, self.missing_data_imputations)
-
+                                  self.holdout_data.copy(deep=True),
+                                  self.treatment_column_name, self.weight_array,
+                                  self.outcome_column_name, self.adaptive_weights, self.alpha,
+                                  self.repeats, self.verbose, self.want_pe,
+                                  self.early_stop_iterations, self.stop_unmatched_c,
+                                  self.early_stop_un_c_frac, self.stop_unmatched_t,
+                                  self.early_stop_un_t_frac, self.early_stop_pe,
+                                  self.early_stop_pe_frac, self.want_bf,
+                                  self.missing_indicator,
+                                  self.missing_data_replace, self.missing_holdout_replace,
+                                  self.missing_holdout_imputations, self.missing_data_imputations)
```

Comment:
My question on this file is that doesn't this suggested change violate the character width? There is a max of 80 characters per line right? Or 180? I don't remember what it is, but just please make sure you're not fixing one issue (indentation level) and creating another issue at the same time :) Feel free to ping me to discuss this one :)

Reply:
Also, this file needs to be synced before merging because I made some more changes to this one today too per Jesse's suggestions.